spec/mongoid/persistable/savable_spec.rb

```diff
@@ -83,6 +83,27 @@
             end
           end
         end
+
+        context 'when the embedded document is unchanged' do
+
+          before do
+            allow(Kangaroo.collection).to receive(:insert).once
```

Comment:
I think it will be better to move expectations and stubs into the actual test block.